package.json

```diff
@@ -25,6 +25,7 @@
   "devDependencies": {
     "babel-eslint": "6.0.0-beta.6",
     "coveralls": "2.11.8",
+    "doctrine": "^1.2.0",
```

Comment:
this would need to be a full `dependency` since it's not related to actually developing the module.